### PR TITLE
[RNMobile] Fix - Only open upload options automatically when the block is inserted

### PR DESCRIPTION
**Pull request by fluiddot**
## Proposed changes:

* The upload options were automatically opened every time the block was selected. Now, similarly to other media blocks, the upload options are only opened when the block was just inserted.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

N/A

## Testing instructions:

1. Tap ➕ to open the block inserter.
2. Add VideoPress block.
3. Observe that the media upload options are opened automatically.
4. Observe that other actions that create the block (i.e. actions that don't imply inserting it from the block inserter) don't open automatically the media upload options.
    - Copy and paste the block.
    - Cut and paste the block.
    - Duplicate the block.
5. Having the VideoPress block in its empty state (showing the placeholder), unselect it by selecting a different block.
6. Tap on the VideoPress block again.
7. Observe that the block is selected and that the media upload options are opened automatically.

### CODE REVIEW

File: projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/index.native.js
Comment (fluiddot) [inline projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/index.native.js:92]: We pass `onFocus` to `MediaPlaceholder` to automatically select the block when the placeholder is tapped ([reference](https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-editor/src/components/media-placeholder/index.native.js#L184-L187)).

File: projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx
Comment (fluiddot) [inline projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx:60]: This condition matches what we have in other media blocks ([reference](https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-library/src/video/edit.native.js#L247-L249)).

**Review by nunyvega — COMMENTED** [review]
Code changes look good to me. Let's wait for a review from people that can test it to give it a green light.

**Review by jhnstn — DISMISSED** [review]
Changes look good and tests well, nice work!

**Comment by fluiddot** [comment]
> Changes look good and tests well, nice work!

@jhnstn I updated the branch with `trunk` and look like after solving the conflicts it dismissed your approval 😞. I'm wondering if you could take a quick look again, thanks 🙇 !

**Review by jhnstn — APPROVED** [review]

**Pull request merged**
